.github/actions/third-party-tests/action.yml

```diff
@@ -0,0 +1,59 @@
+name: 'Test a third party library'
+inputs:
+  lib:
+    type: string
+    required: true
+  version:
+    type: string
+    default: 'latest'
+  network:
+    type: string
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/setup-node@v4
+      with:
+        node-version: '18'
```

Comment:
node 20 is the current LTS. 18 is in maintainance

Reply:
I made it this way that the action can grab the cache from `ci.yml` which is using 18. We could change both at once when we want that